package.json

```diff
@@ -80,8 +92,7 @@
                 2,
                 "ignorePackages",
                 {
-                    "js": "never",
-                    "vue": "never"
```

Comment:
ani `.vue` nechceme ci ?

Reply:
Musel som vyhodit kvoli rollup libke, ta nepozna subory bez '.vue' koncovky bohuzial

Reply:
js určite nechaj
vue pozri či sa to nedá nastaviť rollup